projects/Mallard/src/helpers/settings/defaults.ts

```diff
@@ -1,6 +1,7 @@
-import { Settings } from '../settings'
+import remoteConfig from '@react-native-firebase/remote-config'
```

Comment:
Probably this causing tests to fail:

`remoteConfig is not a function`

Reply:
Just fixed the tests now.